commit 6ca526df40590ce9d52f42c68c58826b1a361bf0
Author: Trent Mick <trentm@gmail.com>
Date:   2019-06-17T13:51:55-07:00 (4 months ago)
    
    OS-7841 remove datasets.joyent.com usage from imgadm test suite (missing update to manifest)
    Reviewed by: Todd Whiteman <todd.whiteman@joyent.com>
    Approved by: Todd Whiteman <todd.whiteman@joyent.com>

diff --git a/src/manifest b/src/manifest
index 97e0ea00..7c096579 100644
--- a/src/manifest
+++ b/src/manifest
@@ -427,7 +427,6 @@ f usr/img/test/IMG.test.js 0444 root bin
 f usr/img/test/basics.test.js 0444 root bin
 f usr/img/test/create.test.js 0444 root bin
 f usr/img/test/docker.test.js 0444 root bin
-f usr/img/test/dsapi.test.js 0444 root bin
 f usr/img/test/import.test.js 0444 root bin
 f usr/img/test/mk-custom-image 0555 root bin
 f usr/img/test/runtests 0555 root bin
